src/main/kotlin/nmcp/internal/task/NmcpZip.kt

```diff
@@ -0,0 +1,5 @@
+package nmcp.internal.task
+
+import org.gradle.api.tasks.bundling.Zip
+
+internal abstract class NmcpZip: Zip()
```

Comment:
Is this class ever used?

Reply:
Nope, this is a leftover, I'll remove it, thanks!